FWCore/Framework/interface/PrincipalGetAdapter.h

```diff
@@ -235,7 +236,7 @@ namespace edm {
     ModuleDescription const& md_;
     
     EDConsumerBase const* consumer_;
-    SharedResourcesAcquirer* resourcesAcquirer_;
+    mutable SharedResourcesAcquirer* resourcesAcquirer_;
```

Comment:
@Dr15Jones The mutability of pointers to the SharedResourcesAcquirer will be reverted in my next commit.